code/modules/clothing/spacesuits/rig_subtypes/rig_subtypes.dm

```diff
@@ -638,6 +638,53 @@
 	species_fit = list(INSECT_SHAPED, GREY_SHAPED)
 	species_restricted = list("exclude", VOX_SHAPED, MUSHROOM_SHAPED)
 
+//Space hobo stuff
+/obj/item/clothing/head/helmet/space/hobo
+	name = "tattered space-proof helmet"
```

Comment:
why would you paste all of this instead of just making a child of the ghetto suit? you only changed 1 stat, the rest would be inherited

Reply:
plans for new spritework i suppose

Reply:
you can still override the sprite without all of this paste

Reply:
okay fine, done